File: Unique-Finance-Tracker/scratch/add_phone_column.py
import sqlite3

db_path = 'c:\\Users\\user\\Downloads\\Unique-Finance-Tracker\\Unique-Finance-Tracker\\sqlite.db'
conn = sqlite3.connect(db_path)
cursor = conn.cursor()

try:
    cursor.execute('ALTER TABLE users ADD COLUMN phone_number TEXT')
    conn.commit()
    print('Added phone_number column to users table.')
except sqlite3.OperationalError as e:
    if 'duplicate column name' in str(e):
        print('Column phone_number already exists.')
    else:
        print(f'Error: {e}')
finally:
    conn.close()

File: Unique-Finance-Tracker/scratch/add_phone_column.mjs
import Database from 'better-sqlite3';
import path from 'path';

const dbPath = 'C:\\Users\\user\\Downloads\\Unique-Finance-Tracker\\Unique-Finance-Tracker\\sqlite.db';
const db = new Database(dbPath);

try {
    db.prepare('ALTER TABLE users ADD COLUMN phone_number TEXT').run();
    console.log('Added phone_number column to users table.');
} catch (err) {
    if (err.message.includes('duplicate column name')) {
        console.log('Column phone_number already exists.');
    } else {
        console.error('Error adding column:', err);
    }
} finally {
    db.close();
}
